deps/yoml/yoml-parser.h

```diff
@@ -33,6 +33,25 @@ extern "C" {
 #include <yaml.h>
 #include "yoml.h"
 
+#define YOML_PARSER_TAG_RESOLVER_INITIAL_STACK_SIZE 4
+
+typedef struct st_yoml_tag_resolver_stack_t {
```

Comment:
use stack as a memo of filenames to detect circular reference
One potential issue: Right now this implementation distinguishes absolute and relative paths, and it may be a bit wasted, but I believe it's not critical issue.